src/table/src/requests.rs

```diff
@@ -212,6 +214,33 @@ pub enum AlterKind {
     RenameTable {
         new_table_name: String,
     },
+    ChangeTableOptions {
+        options: Vec<ChangeTableOptionRequest>,
+    },
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum ChangeTableOptionRequest {
+    TTL(Duration),
```

Comment:
```suggestion
    Ttl(Duration),
```

Reply:
> 

I prefer `TTL`, it's not a word, just an abbreviation.

Reply:
Well, in Rust we usually treat it as a word in naming, like `Arc`. But this is not a blocking issue.